### PR TITLE
Add OT changes section to Explainer

### PR DESCRIPTION
**Pull request by zolkis**
Signed-off-by: Zoltan Kis <zoltan.kis@intel.com>

### CODE REVIEW

File: EXPLAINER.md [hunk 1]
```diff
@@ -258,6 +258,35 @@ From speaking about the feature at multiple events over the years, it is clear
 that web developers have been quite positive about the ability to use NFC from
 the web.
 
+## Changes done after the Chrome Origin Trials (OT)
```
Comment (beaufortfrancois) [inline EXPLAINER.md]:
```suggestion
## Changes done after Origin Trial (OT)

File: EXPLAINER.md [hunk 2]
```diff
@@ -258,6 +258,35 @@ From speaking about the feature at multiple events over the years, it is clear
...
+
+### Replaced `onerror` with `onreadingerror`
+
+Since the merge of `NDEFWriter` and `NDEFReader`, the former `onerror` event on
+`NDEFReader` still relates to scans, but it could be interpreted to be also
+relevant on writes, which is not the case. This change clarifies the error is
+only relevant for scans.
+
+
+These changes mark the Chrome Origin Trial complete.
```
Comment (beaufortfrancois) [inline EXPLAINER.md]: Link to https://developers.chrome.com/origintrials/#/view_trial/236438980436951041

File: EXPLAINER.md [hunk 3]
```diff
@@ -258,6 +258,35 @@ From speaking about the feature at multiple events over the years, it is clear
...
+### Merged NDEFWriter into NDEFReader
+
+Once scan filtering was removed, there was no point in keeping separate objects
+for reading and writing NDEF tags. Hence `NDEFWriter.write` became
+`NDEFReader.write`.
+
+### Removed NDEF write option for ignoring read
+
+When a tag is written, it is also read. With merging the reading and writing
+objects, this option became unneeded. There is an example included on how to
```
Comment (beaufortfrancois) [inline EXPLAINER.md]: Shall we link to https://w3c.github.io/web-nfc/#handling-initial-reads-while-writing

**Review by beaufortfrancois — APPROVED** [review]
LGTM with nits

File: EXPLAINER.md (hunk 2 above)
Comment (zolkis) [reply EXPLAINER.md]: done

File: EXPLAINER.md (hunk 3 above)
Comment (zolkis) [reply EXPLAINER.md]: done

File: EXPLAINER.md [hunk 4]
```diff
@@ -258,6 +258,34 @@ From speaking about the feature at multiple events over the years, it is clear
 that web developers have been quite positive about the ability to use NFC from
 the web.
 
+## Changes done after Origin Trials (OT)
```
Comment (beaufortfrancois) [inline EXPLAINER.md]:
```suggestion
## Changes done after Origin Trial (OT)

File: EXPLAINER.md [hunk 5]
```diff
@@ -258,6 +258,34 @@ From speaking about the feature at multiple events over the years, it is clear
...
+
+### Replaced `onerror` with `onreadingerror`
+
+Since the merge of `NDEFWriter` and `NDEFReader`, the former `onerror` event on
+`NDEFReader` still relates to scans, but it could be interpreted to be also
+relevant on writes, which is not the case. This change clarifies the error is
+only relevant for scans.
+
+
+These changes mark the [Chrome Origin Trials](https://developers.chrome.com/origintrials/#/view_trial/236438980436951041) complete.
```
Comment (beaufortfrancois) [inline EXPLAINER.md]:
```suggestion
These changes mark the [Chrome Origin Trial](https://developers.chrome.com/origintrials/#/view_trial/236438980436951041) complete.

File: EXPLAINER.md (hunk 4 above)
Comment (zolkis) [reply EXPLAINER.md]: On the Chrome page its in plural, that's why I used it (confused).

File: EXPLAINER.md (hunk 5 above)
Comment (beaufortfrancois) [reply EXPLAINER.md]: We've tried once, hence "origin trial" for Web NFC: https://developers.chrome.com/origintrials/#/view_trial/236438980436951041

**Pull request merged**